main.py

```diff
@@ -923,9 +923,7 @@ def showDataInTerminal(
 
         layouts, _ = getTopScores(layouts, scores, SHOW_TOP_LAYOUTS)
         del scores
-        layouts = list(layouts)
-        layouts.reverse()
-        for idx, layout in enumerate(layouts):
+        for idx, layout in enumerate(reversed(layouts)):
```

Comment:
Good job.
I was going to  complain, but then I noticed that the redundant code actually came from me and you fixed it, insted of the other way around.